src/twisted/mail/imap4.py

```diff
@@ -5711,9 +5711,18 @@ def __bytes__(self) -> bytes:
             elif self.empty:
                 base += b"[" + part + b"]"
             if self.partialBegin is not None:
-                base += b"<%d.%d>" % (self.partialBegin, self.partialLength)  # type: ignore[unreachable]
+                if length is None or length > self.partialLength:  # type: ignore[unreachable]
+                    base += b"<%d.%d>" % (self.partialBegin, self.partialLength)
+                else:
+                    # IMAP4rev1 says that if the partial length is greater than
```

Comment:
Can we have a link to the RFC section for this use case?

Reply:
Added